pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "kafe2"
+description = "Karlsruhe Fit Environment 2: a package for fitting and elementary data analysis"
+authors = [
+	{ name="Daniel Savoiu", email="daniel.savoiu@cern.ch" },
+	{ name="Johannes Gäßler", email="johannes.gaessler@cern.ch" },
+	{ name="Cedric Verstege", email="cedric.verstege@cern.ch" },
+	{ name="Günter Quast", email="G.Quast@kit.edu" },
+]
+readme = "README.rst"
+requires-python= ">=3.6"
+keywords = ["kafe2", "kit", "karlsruhe", "data", "analysis", "lab", "laboratory", "practical courses", "education", "university", "students", "physics", "fitting", "minimization", "minimisation", "regression", "parametric", "parameter", "estimation", "optimization", "optimisation"]
+
+license = {text = "GPL3"}
+classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Education",
+        "Intended Audience :: Science/Research",
+        "Topic :: Education",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Physics",
+        "Topic :: Scientific/Engineering :: Information Analysis",
+        "Topic :: Scientific/Engineering :: Visualization",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+]
+#TODO dependency versions
+dependencies = [
+        "iminuit",
+        "NumPy",
+        "Numdifftools",
+        "Scipy",
+        "tabulate",
+        "matplotlib",
+        "PyYaml",
+        "six",
+        "funcsigs",
+        "SymPy",
+        "coverage",
+        "sphinx",
+        "sphinx_bootstrap_theme",
+        "mock",
+]
+dynamic = ["version"]
+
+[tool.setuptools.dynamic]
+version = {attr = "kafe2._version_info.__version__"}
+
+[project.urls]
+"Homepage" = "https://github.com/PhiLFitters/kafe2"
```

Comment:
```suggestion
"Homepage" = "https://philfitters.github.io/kafe2/"
```

Reply:
Thanks. I wasn't aware of the fact that this website even existed